src/main/java/it/gov/pagopa/payment/dto/mapper/AuthPaymentRequestMapper.java

```diff
@@ -0,0 +1,41 @@
+package it.gov.pagopa.payment.dto.mapper;
+
+import it.gov.pagopa.payment.dto.qrcode.AuthPaymentDTO;
+import it.gov.pagopa.payment.dto.qrcode.AuthPaymentRequestDTO;
+import it.gov.pagopa.payment.dto.qrcode.AuthPaymentResponseDTO;
+import it.gov.pagopa.payment.model.TransactionInProgress;
+
+public class AuthPaymentRequestMapper {
+
+  public AuthPaymentRequestDTO rewardMap(TransactionInProgress transactionInProgress) {
```

Comment:
write a test for both these 2 methods (see other mapper tests in the repository)